### PR TITLE
Split vit-station related settings

### PR DESCRIPTION
**Pull request by danielSanchezQ**

### CODE REVIEW

File: testing/jormungandr-testing-utils/src/testing/network_builder/settings.rs [hunk 1]
```diff
@@ -5,24 +5,18 @@ use crate::testing::network_builder::{
...
-        NodeConfig, NodeSecret,
+        ActiveSlotCoefficient, Bft, Block0Configuration, BlockchainConfiguration, CommitteeIdDef,
+        GenesisPraos, Initial, InitialUTxO, LegacyUTxO, NodeConfig, NodeSecret,
     },
 };
 use rand_core::{CryptoRng, RngCore};
 use std::collections::{HashMap, HashSet};
 use std::net::SocketAddr;
-use vit_servicing_station_lib::server::settings::ServiceSettings;
+// use vit_servicing_station_lib::server::settings::ServiceSettings;
```
Comment (mzabaluev) [inline testing/jormungandr-testing-utils/src/testing/network_builder/settings.rs:19]: Just remove this.

Comment (danielSanchezQ) [reply testing/jormungandr-testing-utils/src/testing/network_builder/settings.rs:19]: Leftovers 😞

File: testing/jormungandr-scenario-tests/src/scenario/settings.rs [hunk 2]
```diff
@@ -51,6 +59,72 @@ pub trait PrepareSettings {
         RNG: RngCore + CryptoRng;
 }
 
+#[derive(Debug)]
+pub struct Settings {
+    pub network_settings: NetworkBuilderSettings,
+    pub vit_stations: HashMap<NodeAlias, ServiceSettings>,
+}
+
+impl Deref for Settings {
```
[the remark this answers is not in the document]
Comment (danielSanchezQ) [reply testing/jormungandr-scenario-tests/src/scenario/settings.rs]: And here I was thinking I saved the world 😞

**Review by mzabaluev — APPROVED** [review]
Looks good. I'll rename the struct member as `node_settings` and merge.

**Comment by mzabaluev** [comment]
> I'll rename the struct member as `node_settings`

Or maybe not, it's not only about nodes.

**Pull request merged**